website/docs/dbt-cli/install/pip.md

```diff
@@ -9,11 +10,14 @@ dbt Core and plugins are Python modules distributed on [PyPi](https://pypi.org/p
 <FAQ src="install-pip-best-practices" />
```

Comment:
@jtcohen6 To decrease the number of FAQs on the FAQ page, what do you think about adding the prerequisites and best practices as sections in this doc instead of FAQs? Or are they referenced in another place?

Reply:
good conversation out of this about what we should do when there's information relevant (and significant!) for many, but not all, users